pkg/agent/openflow/pipeline.go

```diff
@@ -755,6 +784,27 @@ func (f *featureService) conntrackFlows() []binding.Flow {
 				Done(),
 		)
 	}
+	// If DSR is enabled, traffic working in DSR mode will be in invalid state on ingress Node.
+	// We forward externally originated packets of invalid connections to stagePreRouting to see if it could be
+	// marked as DSR traffic. If not, they will be dropped in DSRServiceMarkTable.
+	if f.enableDSR {
+		flows = append(flows,
+			ConntrackStateTable.ofTable.BuildFlow(priorityHigh).
+				Cookie(cookieID).
+				MatchRegMark(FromGatewayRegMark).
```

Comment:
we are not matching `FromExternalRegMark`? (asking because of how the comment above is formulated)